### ISSUE TITLE
":tab Foo" does not work for user commands

### ISSUE THREAD

**Issue by justinmk** [vim/vim#802]
`:help :tab` mentions:

```
Doesn't work for :diffsplit, :diffpatch, :execute and :normal.
```

But it doesn't mention user commands. It would avoid a lot of redundant  commands, and add flexibility, if `:tab` supported user commands. 

Steps to reproduce:

```
:command! -nargs=1 Foo split <args>
:tab Foo bar
```
- Expected behavior: opens a new buffer named "bar" in a new tab
- Actual behavior: opens a new buffer named "bar" in a new split window

**Comment by brammool** [comment]
Justin Keyes wrote:

> ```
> Doesn't work for :diffsplit, :diffpatch, :execute and :normal.
> ```
> 
> But it doesn't mention user commands. It would avoid a lot of redundant  commands, and add flexibility, if `:tab` supported user commands. 
> 
> Steps to reproduce:
> 
> ```
> :command! -nargs=1 Foo split <args>
> :tab Foo bar
> ```
> - Expected behavior: opens a new buffer named "bar" in a new tab
> - Actual behavior: opens a new buffer named "bar" in a new split window

OK, but this will cause trouble if the user command has several Ex
commands, then we don't know where to apply the modifier.  E.g. when
temporarily opening a scratch bufffer.

We could make the modifiers available in <> form, so that the user
command can apply them where needed.  Possibly just <mods>, to pass on
all modifiers.

## 

Back off man, I'm a scientist.
              -- Peter, Ghostbusters

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\
\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by chrisbra** [comment]
I think this has been fixed with [7.4.1898](https://github.com/vim/vim/releases/tag/v7.4.1898). Closing
